### ISSUE TITLE
Use syntax + scope with background color

### ISSUE THREAD

**Issue by cbancestry** [daniellmb/JavaScript-Scope-Context-Coloring#5]
Seems like you could get the best of both worlds by highlighting scope using background color and maintaining syntax highlighting.

**Comment by katspaugh** [comment]
Indeed! However, background might be a bit annoying. One could have the same basic color for identifiers, but with different luminosity. Like, the deeper is the callback hell, the darker is the color. Or lighter, if the background is white.

**Comment by MattiSG** [comment]
+1, this is clearly how I have had this in mind for months.
